starter-project-backend-g33/src/resources/articles/controller.ts

```diff
@@ -21,6 +21,7 @@ const getArticle = (req: Request, res: Response, next: NextFunction) => {
     .getOne(req.params['id'])
     .then((data: any) => {
       if (!data) {
+        res.status(404)
```

Comment:
I've dropped an update for all the custom errors we throw.

Reply:
I have added the custom errors as per the examples you dropped above.

Reply:
Great, will review soon and merge